manager/flags_test.go

```diff
@@ -0,0 +1,172 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package manager
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUriString(t *testing.T) {
+	tests := [...]struct {
+		in   Uri
+		want string
+	}{
+		{
+			Uri{
+				Key: "gcm",
+			},
+			"gcm",
+		},
+		{
+			Uri{
+				Key: "influxdb",
+				Val: url.URL{
+					Scheme:   "http",
+					Host:     "monitoring-influxdb:8086",
+					RawQuery: "key=val&key2=val2",
+				},
+			},
+			"influxdb:http://monitoring-influxdb:8086?key=val&key2=val2",
+		},
+	}
+	for _, c := range tests {
+		got := c.in.String()
+		if got != c.want {
+			t.Fatalf("%v.String() got %v, wanted %v", c.in, got, c.want)
```

Comment:
For consistency with the rest of the codebase, can you use assert.Equal instead?